src/components/shared/Modal.css

```diff
@@ -1,19 +1,23 @@
 .modal-wrapper {
   position: fixed;
   display: flex;
+  flex-direction: column;
   align-items: center;
+  justify-content: flex-start;
   width: 100%;
   height: 100%;
   top: 0;
   left: 0;
+  padding-top: 30px;
   transition: z-index 200ms;
   z-index: 100;
 }
 
 .modal {
   top: 0;
-  margin: auto;
   width: 500px;
+  max-height: 40vh;
```

Comment:
im a bit torn here because i don't think `40vh` is a good solution in this case. when the panel is horizontal, it will be really squished:

<img width="1425" alt="screen shot 2017-10-04 at 09 27 52" src="https://user-images.githubusercontent.com/26968615/31164205-5bf4882c-a8e6-11e7-8289-e627a99d1c08.png">

i would instead recommend that we do a media query for horizontal layout, and override with the vertical layout, like so:

```css

  .modal { // use the smallest screen size as the default size, and override from there
    height: 100%;
    overflow-y: scroll;
  }
@media (min-height: 680px) {
  .modal {
    height: auto; // override height at 100% and prefer content height
    max-height: 80vh; // this is a max height, if the content is too large
  }
}
```